plugins/modules/discovery_intent.py

```diff
@@ -70,9 +70,35 @@
       http_read_credential:
         description: HTTP read credentials for hosting a device
         type: dict
+        suboptions:
+            password:
+                description: HTTP(S) password and is mandatory for using HTTP credentials.
+                type: str
+            port:
+                description: HTTP(S) port and is mandatory for using HTTP credentials.
```

Comment:
description: The HTTP(S) port number, which is mandatory for using HTTP credentials.
(or)
description: This is the HTTP(S) port. It is mandatory for the usage of HTTP credentials

Reply:
Addressed